pkg/sql/rename_table.go

```diff
@@ -232,6 +233,29 @@ func (n *renameTableNode) startExec(params runParams) error {
 		return err
 	}
 
+	// If this table has row level ttl enabled, update the schedule_name of all
+	// row-level-ttl jobs with the name table name.
+	if tableDesc.HasRowLevelTTL() {
+		const (
+			opname = `update ttl job names`
+			query  = `update system.scheduled_jobs set schedule_name = 'row-level-ttl-' || $1 WHERE schedule_id = $2`
+		)
+		// TODO Should the number of modified rows be inspected here? Could log a
```

Comment:
I don't think renaming a table should handle this. It's part of what the TTL infra itself should check. cc @ecwall to see if there is a strong guarantee here.

Reply:
Okay! I've removed this comment. Happy to revisit if Evan has opinions.